deliverables.qmd

```diff
@@ -1,9 +1,50 @@
 
 ## What to hand in {#sec-deliv}
 Fork [this template repository](https://github.com/hanneoberman/markup_deliverables/tree/dev) following the flow below. You'll end up with a copy of the repository template on your GitHub account, with the full course exercise and deliverable structure in the `dev` branch. The full rationale about GitHub and branching will be explained in more detail in this course. 
 
-![](images/deliv.gif)
\ No newline at end of file
+![](images/deliv.gif)
+
+## Grading {#sec-grade}
+The following list represents the rubric categories for the deliverables in this course. It will tell you exactly what you will be graded on. All individual deliverables must be completed, submitted, passed or be graded $\geq 5.5$ to pass the course. No compensation is allowed between the different graded deliverables. 
+
+- **1. Presentation of the portfolio (Pass/Fail)**  
+  - All necessary links AND code are presented
+  - All exercises are handed in
+  
+- **2. Personal Repository / Development Repository (Pass/Fail)** 
+  - HINT: don't include your personal phone number in any online representation and check if all links are actually working. 
+  
+- **3. Markup Manuscript (graded; > 5.5; weight: 1/3)**
+  - Use of markup language(s)
+  - Use of figures
+  - Use of tables
+  - Use of equations
+  - Referencing
+  - Overall document presentation and consistency
+  - Alignment of end-result with course aims on [OSIRIS](https://osiris-student.uu.nl/#/onderwijscatalogus/extern/cursus?cursuscode=202000010&taal=en&collegejaar=huidig)
+  - HINT: are your sources included and can the document be compiled from source?
+- **4. Research Compendium (graded; > 5.5; weight: 1/3)**                   
+  - Reproducibility
+  - Structure
+  - Licensing
+  - Code flow and readability
+  - Compilation and robustness
+  - Alignment of end-result with course aims on [OSIRIS](https://osiris-student.uu.nl/#/onderwijscatalogus/extern/cursus?cursuscode=202000010&taal=en&collegejaar=huidig)
+- **5. Shiny app or R-package (graded; > 5.5; weight: 1/3)**
+
+  - Functionality 
+  - Code flow and readability
+  - Compilation and code robustness
+  - Publication / Reference
+  - Development workflow and version control
+  - Alignment of end-result with course aims on [OSIRIS](https://osiris-student.uu.nl/#/onderwijscatalogus/extern/cursus?cursuscode=202000010&taal=en&collegejaar=huidig)
+
+- **6. Make sure it has been submitted on time!**
+
+  - If for some reason this poses a problem for you, please contact the course coordinator as soon as possible.
+  
+You will receive a detailed grading report from us at the end of the course that conforms to the above rubric. Your exact grading, calculated in accordance with the rubric, will be available for review upon request by appointment.
```

Comment:
What is 'a detailed grading report'? Just the full rubric?

Reply:
The margins of the rubric. If you agree, then I'll change that in the text